administration/src/cards/pdf/PdfQrCodeElement.ts

```diff
@@ -27,6 +29,11 @@ const pdfQrCodeElement: PdfElement<PdfQrCodeElementProps, PdfQrCodeElementRender
     qrCode: qrCode,
   }).toBinary()
 
+  // Log base64 activationCode for development purposes
+  if (qrCode.case === 'dynamicActivationCode' && isDevMode()) {
+    console.log(uint8ArrayToBase64(qrCodeContent))
+  }
+
```

Comment:
This change does not come from this PR, does it?

Reply:
this change should have been merge from the #1148
But no idea what happened. Even you need this log for testing